src/main/java/org/example/springintro/repository/order/OrderItemRepository.java

```diff
@@ -0,0 +1,9 @@
+package org.example.springintro.repository.order;
+
+import java.util.List;
+import org.example.springintro.model.OrderItem;
+import org.springframework.data.jpa.repository.JpaRepository;
+
+public interface OrderItemRepository extends JpaRepository<OrderItem, Long> {
+    List<OrderItem> findByOrderId(Long orderId);
```

Comment:
```suggestion
    List<OrderItem> findByIdAndOrderIdAndUserId(Long orderId);
```

provide query with parameters